lib/GridItem.jsx

```diff
@@ -168,44 +193,38 @@ export default class GridItem extends Component {
     };
 
     // This is used for server rendering.
-    if (this.props.usePercentages) {
-      style.left = perc(pos.left / this.props.containerWidth);
-      style.width = perc(pos.width / this.props.containerWidth);
+    if (usePercentages) {
+      style.left = perc(pos.left / containerWidth);
+      style.width = perc(pos.width / containerWidth);
     }
 
     // CSS Transforms support
-    if (this.props.useCSSTransforms) {
+    if (useCSSTransforms) {
       setTransform(style, [pos.left, pos.top]);
       style.left = null;
       style.top = null;
     }
 
     return style;
-  }
+  },
 
   /**
    * Mix a Draggable instance into a child.
    * @param  {Element} child    Child element.
-   * @param  {Object} position  Position object (pixel values)
    * @return {Element}          Child wrapped in Draggable.
    */
-  mixinDraggable(child: ReactElement, position: Position): ReactElement {
-    var start = typeof position.left === "string" ? undefined : {x: position.left, y: position.top};
+  mixinDraggable(child:ReactElement): ReactElement {
     return (
-      <Draggable
-        start={start}
-        moveOnStartChange={this.props.moveOnStartChange}
-        onStop={this.onDragHandler('onDragStop')}
+      <Draggable.DraggableCore
```

Comment:
This was what I was hoping to do - <DraggableCore> was built intended for this module. Thanks for getting it going!